cinema/views.py

```diff
@@ -21,7 +23,7 @@
     MovieSessionDetailSerializer,
     MovieListSerializer,
     OrderSerializer,
-    OrderListSerializer,
+    OrderListSerializer, MovieImageSerializer,
```

Comment:
Code Style: Please group your imports with parentheses consistently. You have `MovieImageSerializer` on the same line with `OrderListSerializer`. It should be on a new line.